comp/otelcol/otlp/components/exporter/datadogexporter/factory.go

```diff
@@ -180,6 +189,38 @@ func (f *factory) createMetricsExporter(
 	return sf.CreateMetricsExporter(ctx, set, ex)
 }
 
+func (f *factory) consumeStatsPayload(ctx context.Context, wg *sync.WaitGroup, statsIn <-chan []byte, tracerVersion string, agentVersion string, logger *zap.Logger) {
```

Comment:
This is copied from https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/factory.go#L254